constants/fallFest2023Content.ts

```diff
@@ -0,0 +1,95 @@
+import { DescriptionCard } from "~/types/uiComponents";
+
+const header = {
+  titleLine1: "Qiskit Fall Fest 2023",
+  description:
+    "The Qiskit Fall Fest is a collection of quantum computing events on college campuses around the world. Every Fall Fest event is organized and run by the students on each campus, with support from IBM Quantum. Student leaders work alongside IBM to design and plan their event, which are run in October and November. Each event is standalone - only made for the students on that particular campus. The events vary from technical hackathons to beginner-friendly challenges, all with the goal of helping to grow the local community’s skill set.",
+  cta: {
+    label: "2022 Fall Fest events",
+    url: "/events/fall-fest-2022",
+    segment: {
+      cta: "learn",
+      location: "header",
+    },
+  },
+  cardSectionHeading: "About the event:",
+  card: {
+    image: "/images/events/fall-fest/fall-fest-logo-2023.png",
+    title: "Qiskit Fall Fest 2023",
+    description:
+      "The Qiskit Fall Fest is a collection of quantum computing events each held by a university!",
+    location: "",
+    time: "",
+    to: "https://medium.com/qiskit/introducing-the-qiskit-fall-fest-feb8456b557",
+    ctaLabel: "Learn about the Fall Fest",
+    segment: {
+      cta: "qiskit-fall-fest-blog",
+      location: "header",
```

Comment:
I would specify that is the card of the header. Something like

```suggestion
      location: "card-header",
```